### PR TITLE
Fully responive web

### PR DESCRIPTION
**Pull request by abraham-kwizera**
Hello Sir, 

This pull request contains codes regarding: 
1.  fully responsive: 
                - portfolio
                - blog page 
                - login page 
                - password restore page 
                - register page

2. update regarding stands up recommendations 

I will be glad to receive a review of my codes.

Abraham Kwizera.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,3 +1,169 @@
 function toggleMobileMenu(menu) {
     menu.classList.toggle('open');
+}
```
Comment (mansurissa) [inline js/index.js:3]: Delete comented code

File: register.html [hunk 2]
```diff
@@ -56,41 +57,46 @@
...
-    <!-- footer -->
-    <footer>
-        <p> Designed and Built by Abraham Kwizera </p>
-    </footer>
-    <!-- end of footer -->
-    <!-- javascript files -->
-    <script src="./js/index.js"></script>
+            </>
+            <!-- </div> -->
+            <!-- </div> -->
```
Comment (mansurissa) [inline register.html]: Same here delete commented code

**Review by mansurissa — COMMENTED** [review]
Delete comented code

**Review by mansurissa — CHANGES_REQUESTED** [review]
Work on comments provided

File: js/index.js (hunk 1 above)
Comment (abraham-kwizera) [reply js/index.js:3]: Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. 
Yours.

File: register.html (hunk 2 above)
Comment (abraham-kwizera) [reply register.html]: Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. allow me to know if changes are seen. 
Yours.

**Pull request merged**